controllers/tlspolicies_validator.go

```diff
@@ -99,6 +104,22 @@ func (t *TLSPoliciesValidator) isTargetRefsFound(topology *machinery.Topology, p
 	return nil
 }
 
+// isConflict Validates if there's already an older policy with the same target ref
+func (t *TLSPoliciesValidator) isConflict(policies []machinery.Policy, p *kuadrantv1.TLSPolicy) error {
+	conflictingP, ok := lo.Find(policies, func(item machinery.Policy) bool {
+		conflictTLSPolicy := item.(*kuadrantv1.TLSPolicy)
+		return p != conflictTLSPolicy && conflictTLSPolicy.DeletionTimestamp == nil &&
```

Comment:
is this correct? Looks like we are checking two pointers? Would they have the same address?

Reply:
Yes, we don't want the conflicting policy to be itself